src/main.rs

```diff
@@ -30,7 +30,7 @@ const GIT_HOMEPAGE: &str = "https://github.com/cita-cloud/network_p2p";
 
 /// network service
 #[derive(Parser)]
-#[clap(version = "0.1.0", author = "Rivtower Technologies.")]
+#[clap(version = "6.3.3", author = "Rivtower Technologies.")]
```

Comment:
```suggestion
#[clap(version = env!("CARGO_PKG_VERSION"), author = "Rivtower Technologies.")]
```

Reply:
`#[clap(version, author)]`
clap will user info in `Cargo.toml`.